examples/toolkit_showcase/toolkit_showcase.ipynb

```diff
@@ -28,47 +28,12 @@
     "</details>\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #45.        new_top.add_molecule(ligand)
</pre><p>Could instead build up a list of "surviving" molecules and run <code>Topology.from_molecules</code> right at the end on that list.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Brilliant. I've also added comments everywhere, since this function is intended to be an example for users. And I decided the <code>topology_from_pdb</code> function in the previous cell wasn't pulling its weight, so I've replaced it with pulling the OpenMM topology directly from the PDBFixer object. </p>

Reply:
:thumbsupparrot: